### ISSUE TITLE
Starting point for sync when wallet imported/created

### ISSUE THREAD

**Issue by AnimusXCASH** [X-CASH-official/android-wallet#6]
Integration of option to synchronizing blocks from certain height. When users create a wallet or import it, they would have special field where they would enter the starting point. 

Currently all wallets gets synchronized from 0 which consumes time.

**Comment by snakewaypasser** [comment]
Yes, It need change some code in xcash-core to support Blockchain Crop.Now if Import wallet choose height.It will use quick sync by use gethash.bin until to specified height then change to getblock.bin.There will be hava an acceleration, but not skip directly.


If support blockchain crop.It can sync very quickly. 

The wallet sync link to here:
https://github.com/X-CASH-official/xcash-core/blob/master/src/wallet/wallet2.cpp
